arangod/Cluster/ClusterMethods.cpp

```diff
@@ -3261,32 +3247,35 @@ arangodb::Result lockDBServerTransactions(std::string const& backupId,
   return finalRes;
 }
 
-arangodb::Result unlockDBServerTransactions(std::string const& backupId,
+arangodb::Result unlockDBServerTransactions(network::ConnectionPool* pool,
+                                            std::string const& backupId,
                                             std::vector<ServerID> const& lockedServers) {
   using namespace std::chrono;
 
   // Make sure all db servers have the backup with backup Id
-  auto cc = ClusterComm::instance();
-  if (cc == nullptr) {
-    // nullptr happens only during controlled shutdown
-    return arangodb::Result(TRI_ERROR_SHUTTING_DOWN, "Shutting down");
-  }
 
   std::string const url = apiStr + "unlock";
-  VPackBuilder lock;
+  
+  VPackBufferUInt8 body;
+  VPackBuilder lock(body);
   {
     VPackObjectBuilder o(&lock);
     lock.add("id", VPackValue(backupId));
   }
-
-  auto body = std::make_shared<std::string const>(lock.toJson());
-  std::vector<ClusterCommRequest> requests;
+  
+  network::RequestOptions reqOpts;
+  reqOpts.retryNotFound = false;
```

Comment:
`retryNotFound` is false by default, so it doesn't need to be set here.
If set here for verboseness, it should be set in the other cases above as well.